### DISCUSSION TITLE
Parenthetical fields?

### DISCUSSION THREAD

**Discussion by mlissner** [freelawproject/courtlistener#3813] [Q&A]
A client writes:

> I'm a little confused by some of the fields in the csv. "described_opinion_id" and "describing_opinion_id" are rather obvious, but what is "group_id"? I know that many of the parentheticals are grouped by some sort of similarity, so is this what this id refers to?
> 
> In addition, what does the "score" value represent?

**Comment by mlissner** [comment]
We get a lot of parentheticals that are nearly identical. When we do, we group them together and all the similar parentheticals get the same `group_id` value. You can see this in action when you look at our website, where things are grouped together: 

![image](https://github.com/freelawproject/courtlistener/assets/236970/e92039b8-4e69-4ffe-b844-5507c4360a9c)

The `score` is how we order the summaries on our site. It's based on a pretty simple scoring system so that the best parentheticals go to the top of the page. The best way to understand the score is to read the code: 

https://github.com/freelawproject/courtlistener/blob/c38845e4b27502c67644e501fa83468d98fcdc8e/cl/citations/score_parentheticals.py

**Reply by amitnagar** [reply]
Sorry but not obvious to me - what is the difference between "described_opinion_id" and "describing_opinion_id" in a parenthetical.

My assumption - The opinion being summarized by the parenthetical is the **described opinion**.

If the above is correct then what does the **describing opinion** refer to? 

Thanks

**Reply by mlissner** [reply]
Well, when an opinion describes another, the one doing the describing is the `describing_opinion_id`, and the one being described is the `described_opinion_id`. Does that help?

**Reply by amitnagar** [reply]
Thanks Michael.

So in the below example the summary is for opinion 1273120 OR 4643569?

id,text,score,described_opinion_id,describing_opinion_id,group_id
"2618061","“The imposition of lifetime parole is not tantamount to a sentence of life imprisonment.”","0.464","1273120","4643569","2287586"

Thank You.

**Reply by mlissner** [reply]
Well, it's the summary of the described opinion, as I said, and the described opinion is 1273120.

**Reply by amitxenai** [reply]
It is clear now. 
Thanks for the patience.
